tests/test_reader.py

```diff
@@ -429,8 +429,8 @@ def test_point():
         assert pt.band_names == ["b1"]
 
         pt = reader.point(src_dst, [310000, 4100000], coord_crs=src_dst.crs)
-        assert pt.data == numpy.array([8917])
-        assert pt.band_names == ["b1"]
+        numpy.testing.assert_equal(pt.data, [8917, 1001])
+        assert pt.band_names == ["b1", "b2"]
```

Comment:
🤔 why do we get a 2 bands dataset when the input COG as only one band?

Reply:
I adjusted the fixture COG (tests/fixtures/cog_scale.tif) to include a second band with the unscaled values and no scale/offset metadata set. In the call above it doesn't pass `indexes=1` like the test above, so it uses both bands in the file.